0.0.6

```diff
@@ -74,14 +74,14 @@
             "description": "Add Type field to AC Task Layout",
             "taskList": [
                 {
-                    "jiraTicket": "ac-57a",
+                    "jiraTicket": "ac-57a,ac-1878",
```

Comment:
Not sure that its necessary to have them both in the jiraTicket field. If theres overlap it will be found in one of the tickets when theyre checked.